### ISSUE TITLE
Close StructureMoleculeComponent settings pane on click outside

### ISSUE THREAD

**Issue by janosh** [materialsproject/crystaltoolkit#311]
Just a small QoL improvement suggestion: It feels more natural if settings panes like the one on the `StructureMoleculeComponent` hide when clicking outside rather than having to go up and do a precision click on the close button.

https://user-images.githubusercontent.com/30958850/216792250-5aa1e3b5-cbca-4608-ae31-828af4aafa98.mp4

**Comment by mkhorton** [comment]
Agree. From a usability POV, the partial transparency is not good either. It would require a change in `mp-react-components` to update however. We have also had feedback that the download/export button is not very discoverable, which is unfortunate because I think it's something people would like to use often!

**Comment by janosh** [comment]
> We have also had feedback that the download/export button is not very discoverable, ...

I think so too. Took me a while to find it.
